locales/messages.es.json

```diff
@@ -0,0 +1,72 @@
+  "LicenseExpressionExpectExceptionFoundParen": "Expected an exception name, found a parenthesis.",
+  "LicenseExpressionExpectLicenseFoundCompound": "Expected a license name, found the compound {value}.",
+  "LicenseExpressionExpectLicenseFoundEof": "Expected a license name, found the end of the string.",
+  "LicenseExpressionExpectLicenseFoundParen": "Expected a license name, found a parenthesis.",
+  "LicenseExpressionImbalancedParens": "There was a close parenthesis without an opening parenthesis.",
+  "LicenseExpressionUnknownException": "Unknown license exception identifier '{value}'. Known values are listed at https://spdx.org/licenses/exceptions-index.html",
+  "LicenseExpressionUnknownLicense": "Unknown license identifier '{value}'. Known values are listed at https://spdx.org/licenses/",
+  "NoLocalizationForMessages": "No localization for the following messages:",
+  "ProcessorArchitectureMalformed": "Failed to parse %PROCESSOR_ARCHITECTURE% ({arch}) as a valid CPU architecture.",
+  "ProcessorArchitectureMissing": "The required environment variable %PROCESSOR_ARCHITECTURE% is missing.",
+  "ProcessorArchitectureW6432Malformed": "Failed to parse %PROCESSOR_ARCHITEW6432% ({arch}) as a valid CPU architecture. Falling back to %PROCESSOR_ARCHITECTURE%.",
+  "SeeURL": "See {url} for more information.",
+  "UnsupportedSystemName": "Error: Could not map VCPKG_CMAKE_SYSTEM_NAME '{system_name}' to a vcvarsall platform. Supported system names are '', 'Windows' and 'WindowsStore'.",
+  "UnsupportedToolchain": "Error: in triplet {triplet}: Unable to find a valid toolchain combination.\n    The requested target architecture was {arch}\n    The selected Visual Studio instance is at {path}\n    The available toolchain combinations are {list}\n",
+  "UsingManifestAt": "Using manifest file at {path}.",
+  "VSExaminedInstances": "The following Visual Studio instances were considered:",
+  "VSExaminedPaths": "The following paths were examined for Visual Studio instances:",
+  "VSNoInstances": "Could not locate a complete Visual Studio instance",
+  "VcpkgDisallowedClassicMode": "Error: Could not locate a manifest (vcpkg.json) above the current working directory.\nThis vcpkg distribution does not have a classic mode instance.",
+  "VcpkgHasCrashed": "vcpkg.exe has crashed.\nPlease send an email to:\n    {email}\ncontaining a brief summary of what you were trying to do and the following data blob:",
+  "VcpkgHasCrashedArgument": "{value}|",
+  "VcpkgHasCrashedDataBlob": "\nVersion={version}\nEXCEPTION='{error}'\nCMD=",
+  "VcpkgInvalidCommand": "invalid command: {command_name}",
+  "VcpkgSendMetricsButDisabled": "Warning: passed --sendmetrics, but metrics are disabled.",
+  "VersionSpecMismatch": "error: Failed to load port because version specs did not match\n    Path: {path}\n    Expected: {expected_version}\n    Actual: {actual_version}",
+  "WarningMessage": "warning: "
+}
```

Comment:
Why do some of these have a terminal newline and some do not? (Is that intentional?)

Reply:
none of the messages should have a terminal newline; the comments may or may not depending on implementation

Reply:
I'm talking about this JSON file itself, not any of the individual messages.

Reply:
(Sorry "these" was confusing")

Reply:
Looks like en has one but not the others

Reply:
ah, that's because of the localization software. The stuff that's localized does not contain a newline